operations/research-and-development/product/release-process/release-overview.md

```diff
@@ -82,15 +82,24 @@ Release dates are currently communicated in the following ways. Further iteratio
 
 Release Manager is able to look at the version we have deployed to Mattermost Cloud. Also, when we are hooked up to split.io, there will be a dashboard where you can see active and historic feature flags. Any feature that has its flag removed would be included on any Self-Managed releases past the point after that removal was merged. More details on feature flags: https://developers.mattermost.com/contribute/server/feature-flags/.
 
+## Cloud Release Branch Processes
+
+**Process for merging the ``master`` branch into the ``cloud`` branch**
+ - The Cloud team opens PRs in webapp, server, redux, api-reference, and Enterprise branches to merge the ``master`` branch into the ``cloud`` branch ([example PR](https://github.com/mattermost/mattermost-server/pull/16838)).
```

Comment:
Should "Enterprise" include a capital here? or should it be "enterprise"?

Reply:
@cwarnermm I capitalized it based on Justine's suggestion https://github.com/mattermost/mattermost-handbook/pull/629#discussion_r582202940 - I'm 0/5.